pkg/controller/status/controller.go

```diff
@@ -91,32 +92,32 @@ func StartFederationStatusController(controllerConfig *util.ControllerConfig, st
 }
 
 // newFederationStatusController returns a new status controller for the federated type
-func newFederationStatusController(controllerConfig *util.ControllerConfig, typeConfig typeconfig.Interface) (*FederationStatusController, error) {
+func newFederationStatusController(config *util.ControllerConfig, typeConfig typeconfig.Interface) (*FederationStatusController, error) {
```

Comment:
(No action required) What is your thinking behind s/controllerConfig/config/?  Config is such a generic term that I used `controllerConfig` to eliminate potential ambiguity.

Reply:
Was trying to be coherent. I know these changes should be ideally separate as you said. would try to conform in future.